src/configure

```diff
@@ -1309,18 +1309,32 @@ elif [ "`uname`" == "Linux" ]; then
     if [ -z "$OPENBLASROOT" ]; then
       failure "Must specify the location of OPENBLAS with --openblas-root option (and it must exist)"
     fi
-    if [ ! -f $OPENBLASROOT/lib/libopenblas.so ]; then
+    if [ -f $OPENBLASROOT/lib/libopenblas.so ]; then
+      OPENBLASLIBDIR=$OPENBLASROOT/lib
+    elif [ -f $OPENBLASROOT/lib64/libopenblas.so ]; then
+      # in REDHAT/CentOS package installs, the library is located here
+      OPENBLASLIBDIR=$OPENBLASROOT/lib64
+    else
       failure "Expected to find the file $OPENBLASROOT/lib/libopenblas.so"
     fi
+    if [ -f $OPENBLASROOT/include/cblas.h ] ; then
+      OPENBLASINCDIR=$OPENBLASROOT/include
+    elif [ -f $OPENBLASROOT/include/openblas/cblas.h ] ; then
+      # in REDHAT/CentOS/Ubuntu package installs, the includes are located here
+      OPENBLASINCDIR=$OPENBLASROOT/include/openblas
+    else 
+      echo "Assuming openblas is aleady in a default include path"
```

Comment:
You should probably print a scarier warning here, e.g.
echo "$0: ***** Using OpenBlas from $OPENBLASROOT but cblas.h is not found.  "
echo "     ******  Assuming openblas is aleady in a default include path, but"
echo "      *****  if you get compilation messages about not finding files like cblas.h,"
echo "      ***** you should look into this (e.g. make sure to install the 'openblas-dev' package,"
echo "      ***** if it is a package-based install)."


Also, please confirm that you tested that this still works if you use the default OPENBLASROOT based on the install from tools/.

Reply:
Incorporated.  And yes I am testing this with the tools openblas.